client/components/main/Desktop/CouncilSelector/index.js

```diff
@@ -1,7 +1,11 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { updateNcId } from '@reducers/filters';
+import { connect, useDispatch, useSelector } from 'react-redux';
```

Comment:
Should Remove useDispatch and useSelect if not being used.

Reply:
Thanks. Please try again. Everything should be updated.